src/hooks/usePastTransaction/usePastTransaction.test.tsx

```diff
@@ -10,7 +10,14 @@ import { defaultIdentifier } from "../../test/helpers/defaults";
 import { ProductContextProvider } from "../../context/products";
 import { ERROR_MESSAGE } from "../../context/alert";
 
-jest.mock("../../services/quota");
+jest.mock("../../services/quota", () => {
+  const actualModule = jest.requireActual("../../services/quota");
+  return {
+    ...actualModule,
+    // Only mock
+    getPastTransactions: jest.fn()
+  };
+});
```

Comment:
In the final test case where we test if `PAST_TRANSACTION_ERROR` is thrown, previously it was simply throwing an `Error` type with `PAST_TRANSACTION_ERROR` message.

I have since changed it to throw `PastTransactionError` which is under the same module as `../../services/quota`, which is mocked. Since it is mocked, we cannot assign an actual value to the `PastTransactionError`'s message unless we get the actual implementation.

Let me know if there is a better way we can do this!

Reply:
Ah I understand what you are trying to do now, thanks for explaining!